doc/source/examples/dem/granular-mixer/granular-mixer.rst

```diff
@@ -0,0 +1,144 @@
+==================================
+Granular Mixer
+==================================
+
+This example simulates the packing and mixing of particles in a mixer with a pitched-blade impeller. It is recommended to visit `DEM parameters <../../../parameters/dem/dem.html>`_ for more detailed information on the concepts and physical meanings of the parameters in Lethe-DEM.
+
+Features
+----------------------------------
+- Solvers: ``dem_3d``
+- Floating mesh
+- Gmsh grids
+- Bidispersed particles
+
+
+Location of the examples
+------------------------
+ ``/examples/dem/3d_granular_mixer/granular_mixer.prm``
+
+
+Description of the case
+-----------------------
+
+This simulation consists of two stages: packing (0-0.5 s) and mixing (0.5-5 s). There are two types of particles in this simulation (bidispersed system), that are inserted on top of each other during the packing stage. At :math:`t=0.5` s, the pitched-blade impeller starts rotating with an angular velocity of 6 rad/s and mixes the particles.
```

Comment:
```suggestion
This simulation consists of two stages: packing (0-0.5 s) and mixing (0.5-5 s) of particles. There are two types of particles in this simulation (with a respective diameter of X and Y), that are inserted on top of each other during the packing stage. At :math:`t=0.5` s, the pitched-blade impeller starts rotating with an angular velocity of 6 rad/s and mixes the particles.
```
I would specify the diameter of the particles directly

Reply:
Ok now I just understood the have the same size. I would explain that in the context and make that clear from the get go that we have two particle type only to measure the mixing.

Reply:
Added to the case description.